core/src/main/scala/zio/logging/Logging.scala

```diff
@@ -1,45 +1,95 @@
 package zio.logging
 
+import zio._
 import zio.clock.{ currentDateTime, Clock }
 import zio.console.{ putStrLn, Console }
-import zio._
 
 object Logging {
-  type Logging = Has[Logging.Service]
-
-  trait Service {
-    def logger: Logger
-  }
+  type Logging = Has[Logger[String]]
 
   def console(
-    format: (LogContext, => String) => String
+    format: (LogContext, => String) => String,
+    rootLoggerName: Option[String] = None
   ): ZLayer[Console with Clock, Nothing, Logging] =
-    make((context, line) =>
-      for {
-        date       <- currentDateTime.orDie
-        level      = context(LogAnnotation.Level)
-        loggerName = context(LogAnnotation.Name)
-        maybeError = context
-          .get(LogAnnotation.Throwable)
-          .map(Cause.fail)
-          .orElse(context.get(LogAnnotation.Cause))
-          .map(cause => System.lineSeparator() + cause.prettyPrint)
-          .getOrElse("")
-        _ <- putStrLn(date.toString + " " + level + " " + loggerName + " " + format(context, line) + " " + maybeError)
-      } yield ()
+    make(
+      (context, line) =>
+        for {
+          date       <- currentDateTime.orDie
+          level      = context(LogAnnotation.Level)
+          loggerName = context(LogAnnotation.Name)
+          maybeError = context
+            .get(LogAnnotation.Throwable)
+            .map(Cause.fail)
+            .orElse(context.get(LogAnnotation.Cause))
+            .map(cause => System.lineSeparator() + cause.prettyPrint)
+            .getOrElse("")
+          _ <- putStrLn(date.toString + " " + level + " " + loggerName + " " + format(context, line) + " " + maybeError)
+        } yield (),
+      rootLoggerName
     )
 
+  val context: URIO[Logging, LogContext] =
+    ZIO.accessM[Logging](_.get.logContext)
+
+  def debug(line: => String): ZIO[Logging, Nothing, Unit] =
+    ZIO.accessM[Logging](_.get.debug(line))
+
+  def error(line: => String): ZIO[Logging, Nothing, Unit] =
+    ZIO.accessM[Logging](_.get.error(line))
+
+  def error(line: => String, cause: Cause[Any]): ZIO[Logging, Nothing, Unit] =
```

Comment:
What do you think about reverting to the logError naming scheme? I think it will be quite common for people to do a wildcard import zio.logging.Logging._ (would look nicer if these were in the package object imo), but with the current naming this will probably cause name clashes.

The alternative of having to log everywhere with Logging.info("bla") seems too verbose

Reply:
this is good idea. I will keep `log.debug` and here we will have `logDebug`. @jdegoes what do you think?

Reply:
I don't think someone will import `Logging._` because it contains all the constructors.

Maybe keeping the `log` helper is sufficient, e.g. `log.info` reads rather well (in my eyes), at least as well as `logInfo`.

Reply:
in my opinion ```log.info``` looks better. I'm not going to change this for now